addons/website_hr_recruitment/tests/test_website_hr_recruitment.py

```diff
@@ -7,17 +7,29 @@
 @odoo.tests.tagged('post_install', '-at_install')
 class TestWebsiteHrRecruitmentForm(odoo.tests.HttpCase):
     def test_tour(self):
-        job = self.env['hr.job'].create({
-            'name': 'A Test Job',
+        job_guru = self.env['hr.job'].create({
+            'name': 'Guru',
             'is_published': True,
         })
-
+        job_intern = self.env['hr.job'].create({
+            'name': 'Internship',
+            'is_published': True,
+        })
+        self.start_tour('/', 'website_hr_recruitment_tour_edit_form', login='admin')
         with odoo.tests.RecordCapturer(self.env['hr.applicant'], []) as capt:
-            self.start_tour("/", 'website_hr_recruitment_tour')
+            self.start_tour('/', 'website_hr_recruitment_tour')
```

Comment:
@Guillaume-gdi When you fix a conflict, can you try to fix it in one push-force? Here I see two push-force:

- One probably rebasing + removing the conflict without fixing it properly
- One fixing the fact that the conflict was badly resolved

=> I cannot see how you resolved the conflict :shrug: 

Doing that way, you also end up with that kind of useless diff (the line I am commenting here).